projects/openstreetmap-website-notes-refactoring/README.md

```diff
@@ -0,0 +1,36 @@
+# OpenStreetMap-Website: Refactor Notes
+
+The goal of this project is to refactor the existing notes implementation on the OSM website.
+
+* Project: [openstreetmap-website](https://github.com/openstreetmap/openstreetmap-website)
+* Related:
+  - https://github.com/openstreetmap/openstreetmap-website/issues/3831
+  - maybe https://github.com/openstreetmap/openstreetmap-website/pull/3934
+
+The expected deliverable is a mergeable pull request towards https://github.com/openstreetmap/openstreetmap-website. It must at least:
+- Follow the idea of a refactoring by changing as little as possible in behaviour and improving the underlying code structure
+- Handle the UI part – Map, Forms, Lists, …
+- Handle the API part – without any breaking changes to the API
+- Handle the Data migration of existing data – discuss details with the operations team
+- Handle freatures related to moderation and administration
+- Handle special cases for blocked users, hidden or removed notes and similar cases
+- Include a wireframe of all proposed UI changes
+- Conform to the [CONTRIBUTING.md doc](https://github.com/openstreetmap/openstreetmap-website/blob/master/CONTRIBUTING.md)
+
+## Positive effects of this project
+
+**Remove bugs and improve stability:**
+
+- There are situations when the Notes API returns an error 500 right now. This refactoring will solve the underlying issue. There are multiple issues and even more dupliactes on this topic, eg. https://github.com/openstreetmap/openstreetmap-website/issues/2146.
+- It will also solve edgecases like https://github.com/openstreetmap/openstreetmap-website/issues/3396
```

Comment:
Why is it an "also" case?

The first kind of "situations" (with error 500) is a subset of the "edgecases" referred here. If the user is deleted and nobody else did anything with the note, you get error 500. If somebody did something, that somebody gets promoted to note author.

Reply:
@AntonKhorev I change the wording to only reference the issue.